Hermes/appscale/hermes/stats/stats_app.py

```diff
@@ -144,6 +154,101 @@ def get_cluster_stats_api_routes(is_master):
   ]
 
 
+class ProfilingManager(object):
+  def __init__(self, zk_client):
+    self.nodes_profile_log = None
+    self.processes_profile_log = None
+    self.proxies_profile_log = None
+    self.nodes_profile_task = None
+    self.processes_profile_task = None
+    self.proxies_profile_task = None
+
+    def bridge_to_ioloop(update_function):
+      def do_in_ioloop():
+        IOLoop.current().add_callback(update_function)
```

Comment:
Thanks for noticing! No magic here, I just lost a commit which fixes this problem in a different branch=(